OmiseSDK/OmiseJsonParser.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-@objc(OMSOmiseJsonParser) public final class OmiseJsonParser: NSObject {
```

Comment:
I think this should be OMSJSONParser follow the Apple convention `NSJSONSerialization`